src/FakeItEasy.ruleset

```diff
@@ -1,9 +1,11 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <RuleSet Name="FakeItEasy" ToolsVersion="11.0">
   <Include Path="allrules.ruleset" Action="Default" />
   <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">
     <Rule Id="CA1020" Action="None" />
+    <Rule Id="CA1200" Action="None" />

```

Comment:
"Avoid using cref tags with a prefix". Apparently it doesn't like the `<see cref="!:http://..." />` syntax. This isn't a standard syntax, so I think what it actually doesn't like is prefixes such as "M:" (method), "T:" (type) etc; the "!:" prefix was just caught accidentally.